### PR TITLE
Support building tetris sample for MinGW32

### PR DESCRIPTION
**Pull request by ddolovov**

### CODE REVIEW

File: samples/tetris/build.gradle [hunk 1]
```diff
@@ -104,12 +112,27 @@ afterEvaluate {
...
 // If host platform is Linux and RaspberryPi target is activated.
 private boolean isRaspberryPiBuild() {
     MPPTools.isLinux() && Boolean.parseBoolean(project.findProperty('tetris.raspberrypi.build') as String)
-}
\ No newline at end of file
+}
+
+private boolean isMingwX86Build() {
+    MPPTools.isWindows() && (
+            System.getenv("ProgramFiles(x86)") == null
```
Comment (msink) [inline samples/tetris/build.gradle]: Why? Host must be 64-bit, so here cannot be null.

File: samples/tetris/build.gradle [hunk 2]
```diff
@@ -60,10 +62,16 @@ kotlin {
     }
 }
 
-if (hostPreset == kotlin.presets.mingwX64) {
+if ((hostPreset == kotlin.presets.mingwX86) || (hostPreset == kotlin.presets.mingwX64)) {
     tasks.create('compileWindowsResources', Exec) {
+        def windresDir = null
+        if (hostPreset == kotlin.presets.mingwX86)
+            windresDir = "${MPPTools.kotlinNativeDataPath()}/dependencies/msys2-mingw-w64-i686-gcc-7.4.0-clang-llvm-6.0.1/bin"
```
Comment (olonho) [inline samples/tetris/build.gradle]: Ugly :(. Maybe at least smth like `fun targetToolchainPath(hostPreset)`.

File: samples/tetris/build.gradle (hunk 1 above)
Comment (ddolovov) [reply samples/tetris/build.gradle]: 32-bit host does not have `ProgramFiles(x86)` environment variable. That's why 32-bit tetris will be built under 32-bit host. What's not correct here?

File: samples/tetris/build.gradle (hunk 2 above)
Comment (ddolovov) [reply samples/tetris/build.gradle]: Another _bad_ function for _MPPTools_?

Comment (ddolovov) [reply samples/tetris/build.gradle]: I would suggest to add this to K/N Samples Wishlist but leave it as is for now.

File: samples/tetris/build.gradle (hunk 1 above)
Comment (olonho) [reply samples/tetris/build.gradle]: 32-bit Windows hosts cannot run Kotlin/Native compiler.

Comment (msink) [reply samples/tetris/build.gradle]: Kotlin/Native does not support 32-bit hosts.

Comment (ddolovov) [reply samples/tetris/build.gradle]: OK

**Pull request merged**